### PR TITLE
URL embedding

### PR DESCRIPTION
**Pull request by KrishMunot**

### CODE REVIEW

**Comment by KrishMunot** [comment]
@JoelMarcey can you have a look at this too?

**Comment by JoelMarcey** [comment]
@KrishMunot Hi. Anyway you can rebase or somehow resolve the conflicts mentioned above?

File: _docs/1-A2-relay.md [hunk 1]
```diff
@@ -12,8 +12,6 @@ When we originally planned the app [we discussed the choice of data layer](tutor
 
 At the time, Redux was chosen because it offered the simpler data implementation of the two, and was quicker and easier to integrate with our Parse data storage.
 
-Once the app was released to the [iOS](https://itunes.apple.com/us/app/f8/id853467066) and [Android](https://play.google.com/store/apps/details?id=com.facebook.f8) app stores, we wanted to revisit that choice, and look at how Relay could work with our app.
```
Comment (JoelMarcey) [inline _docs/1-A2-relay.md]: @KrishMunot Did you intend to delete this entire paragraph?

Comment (JoelMarcey) [inline _docs/1-A2-relay.md]: Just a friendly ping on my question above.

Comment (KrishMunot) [inline _docs/1-A2-relay.md]: @JoelMarcey That deletion of the paragraph was unintentional and I've fixed it along with another small change.

**Pull request merged**